v2/concordium/types.proto

```diff
@@ -1306,6 +1369,63 @@ message DelegationEvent {
   }
 }
 
+//Token events originating from account transcations.
+message TokenEvent {
```

Comment:
It seems that we don't use `TokenEvent` itself anywhere

Reply:
I understand block special events as events that are not tied to some transaction on chain, like rewards and suspension, the above events are produced from an account transaction so should probably be in `AccountTransactionEffects`.